docs/Guide/CLI/generating-components.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Generating components
+sidebar_position: 2
+---
+
+We can generate components using:
+
+```bash
+sapphire generate <component> <name>
```

Comment:
How does this work with alternative formatting placeholders? Those need more documentation, I feel.